### ISSUE TITLE
The result of checking for updates should respond with a return value

### ISSUE THREAD

**Issue by Geek-Bob** [ravibpatel/AutoUpdater.NET#667]
The result of checking for updates should be a return value response, rather than an event response, because when manually click to check for updates, if there is no new version, you need to make a friendly reminder, but the background periodic check for updates does not need a reminder

**Comment by billfox-lui** [comment]
Yes, it is recommended to add a reminder function, as having no new version available and being unable to connect to the server result in the same outcome - no response at all.

**Comment by huihuiliu2** [comment]
Yes, when manually updating, there is often a problem that the connection is not the server, and hopefully there is a return value accordingly.
